recipes/zephyr-7b-beta/dpo/config_qlora.yaml

```diff
@@ -1,12 +1,12 @@
 # Model arguments
 model_name_or_path: alignment-handbook/zephyr-7b-sft-qlora
-torch_dtype: float16
+torch_dtype: bfloat16
 
 # LoRA arguments
 use_peft: true
 load_in_4bit: true
-lora_r: 16
-lora_alpha: 16
+lora_r: 128
```

Comment:
Tuning these hparams was necessary to get close to `zephyr-7b-beta` perf on MT-Bench